src/components/marketing-location/dto/list-marketing-location.dto.ts

```diff
@@ -0,0 +1,35 @@
+import { Field, InputType, ObjectType } from '@nestjs/graphql';
+import { Type } from 'class-transformer';
+import { ValidateNested } from 'class-validator';
+import { PaginatedList, SortablePaginationInput } from '../../../common';
+import { MarketingLocation } from './marketing-location.dto';
+
+@InputType()
+export abstract class MarketingLocationFilters {
+  @Field({
+    description: 'Only marketing locations matching this name',
+    nullable: true,
+  })
+  readonly name?: string;
```

Comment:
I don't think we need these name filters any more since we are doing them in global search.